wgpu/src/backend/web.rs

```diff
@@ -340,6 +340,18 @@ fn map_primitive_state(primitive: &wgt::PrimitiveState) -> web_sys::GpuPrimitive
     //TODO:
     //mapped.unclipped_depth(primitive.unclipped_depth);
 
+    match primitive.polygon_mode {
+        wgt::PolygonMode::Fill => {}
+        wgt::PolygonMode::Line => panic!(
+            "{:?} is not enabled for this backend",
+            wgt::Features::POLYGON_MODE_LINE
+        ),
+        wgt::PolygonMode::Point => panic!(
+            "{:?} is not enabled for this backend",
+            wgt::Features::POLYGON_MODE_POINT
+        ),
```

Comment:
nitpick: Pushed this suggestion as c2c0e420: 

```suggestion
        mode @ wgt::PolygonMode::Line | wgt::PolygonMode::Point => {
            panic!("{mode:?} is not enabled for this backend",)
        }
```

Reply:
I think the panic should mention the feature rather than the mode itself. There is precedent for this, if we want to use the mode instead we should discuss about it and do it everywhere.

Reply:
Yeah, the feature is the error you get elsewhere as well. We should keep this consistant.

Reply:
Ah, I see how this refactor was incorrect. Sorry about that!